src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -243,8 +252,12 @@ private boolean save(Path targetPath, SaveDatabaseMode mode) {
                 libraryTab.resetChangedProperties();
             }
             dialogService.notify(Localization.lang("Library saved"));
+
+            if (success) {
+                this.automaticGitUpdate(targetPath);
+            }
             return success;
-        } catch (SaveException ex) {
+        } catch (SaveException | IOException | GitAPIException ex) {
             LOGGER.error(String.format("A problem occurred when trying to save the file %s", targetPath), ex);
```

Comment:
Don't use `String.format`. Use `{}`

```suggestion
            LOGGER.error("A problem occurred when trying to save the file {}", targetPath, ex);
```